lib/OpenQA/Worker/Job.pm

```diff
@@ -445,7 +445,11 @@ sub _stop_step_4_upload ($self, $reason, $callback) {
             my @other = (
                 @{find_video_files($pooldir)->map('basename')->to_array},
                 COMMON_RESULT_FILES,
-                qw(serial0 video_time.vtt serial_terminal.txt virtio_console.log virtio_console1.log)
+                # TODO: use $vars->{VIRTIO_CONSOLE_NUM} for virtio names
```

Comment:
Don't add TODOs or disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit or create a ticket for what needs to be done

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Reply:
Well, I don't agree with this policy (there is higher chance that it will be fixed when documented in the code than in the ticket; also here is kind of documentation that it's related to `VIRTIO_CONSOLE_NUM` thus it help people to understand that relation), but sure I'll remove it.